lib/ansible/module_utils/urls.py

```diff
@@ -385,6 +391,21 @@ def as_list(self):
         return [self.get(k, None) for k in ('scheme', 'netloc', 'path', 'params', 'query', 'fragment')]
 
 
+def insecure_tls_options(context):
+    '''
+    Add support for insecure TLS options when requested.
+    This is required when dealing with older devices,
+    or useful when needing to upgrade older devices.
+    '''
+    # TODO: We may want to enable older ciphers here too ?
+    # context.set_ciphers('HIGH:!aNULL:!eNULL')
+    context.options &= ~ssl.OP_NO_SSLv2
```

Comment:
We need to check if `ssl.OP_NO_SSLv2` it not `None` before using it.  Newer versions of openssl have removed it, which may cause issues here. See https://github.com/ansible/ansible/commit/496d10f

Reply:
Makes sense, but then I prefer to compare to None explicitly. Thanks!